src/layer/poi.js

```diff
@@ -7,6 +7,7 @@ var iconDefs = {
   hospital: "hospital",
   medical: ["doctors", "clinic"],
   parking: "parking",
+  school: "school",
```

Comment:
> This PR does not render colleges/universities or preschools.

In the absence of a separate entry for colleges and universities, calling this entry “school” would imply that it also includes colleges and universities. Maybe we should match `college` and `university` at the same time for now, then split them out once we have a distinct icon or treatment for them.

Reply:
Colleges and universities are now included